nats-server/src/lib.rs

```diff
@@ -22,6 +22,8 @@ use std::{thread, time::Duration};
 use lazy_static::lazy_static;
 use rand::Rng;
 use regex::Regex;
+use serde_json as json;
```

Comment:
Might as-well qualify this.

```suggestion
use serde_json;
```

Reply:
Fixed along with a friendlier error message in case the parsing failed.